### PR TITLE
fix(Scripts/boss_freya): adds apply SPELL_CONSERVATOR_GRIP 6s after spawn

### LINKED ISSUE #4083: [Ulduar] [Freya] [Fix included] Ancient Conservator add aura fix
#CURRENT BEHAVIOUR:
Ancient Conservator adds apply their pacifying and silencing aura instantly.
https://streamable.com/5tggk5

EXPECTED BLIZZLIKE BEHAVIOUR:
Ancient Conservator adds apply their pacifying and silencing aura after 6 seconds.
https://www.youtube.com/watch?v=mc2ZOf3CkKA&ab_channel=Ciderhelm

STEPS TO REPRODUCE THE PROBLEM:
.cheat god
Enter Ulduar
Move or teleport to Freya
Pull her, wait for the Ancient Conservator to spawn. Look how the [Conservator's Grip](https://wotlkdb.com/?spell=62532) is applied instantly.

EXTRA NOTES:
https://github.com/azerothcore/azerothcore-wotlk/pull/4082
fixes the problem

AC HASH/COMMIT:
c208a12
from June 3rd

OPERATING SYSTEM:
OS: Ubuntu Server 20.04

MODULES:
none

OTHER CUSTOMIZATIONS:
No customizations at all

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/95569550-ulduar-freya-fix-included-ancient-conservator-add-aura-fix?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by 55Honey**
Ancient Conservator's Grip spell moved to an event and delayed by 6 seconds on spawn to make it blizzlike.

## Changes Proposed:
Ancient Conservator adds apply their pacifying and silencing aura 6 seconds after spawn.

## Issues Addressed:
Ancient Conservator adds apply their pacifying and silencing aura instantly.
fixes #4083 

## Tests Performed:
Tested on Win10 64
https://streamable.com/d6em9x

## How to Test the Changes:
.cheat god
Enter Ulduar
Move or teleport to Freya
Pull her, wait for the Ancient Conservator to spawn. Look how the [Conservator's Grip](https://wotlkdb.com/?spell=62532) is applied instantly without the fix and after 6 seconds with the fix.

## Known Issues and TODO List:
none

## Target Branch(es):
- [x] Master

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: src/server/scripts/Northrend/Ulduar/Ulduar/boss_freya.cpp [hunk 1]
```diff
@@ -1159,7 +1160,7 @@ class boss_freya_summons : public CreatureScript
             if (me->GetEntry() == NPC_ANCIENT_CONSERVATOR)
             {
                 me->CastSpell(me, SPELL_HEALTHY_SPORE_SUMMON, true);
-                me->CastSpell(me, SPELL_CONSERVATOR_GRIP, true);
+				events.ScheduleEvent(EVENT_ANCIENT_CONSERVATOR_GRIP, 6000);
```
Comment (wizzymore) [inline src/server/scripts/Northrend/Ulduar/Ulduar/boss_freya.cpp:1163]:
```suggestion
                events.ScheduleEvent(EVENT_ANCIENT_CONSERVATOR_GRIP, 6000);
```
Change indent to spaces and make it equal to the rest

File: src/server/scripts/Northrend/Ulduar/Ulduar/boss_freya.cpp [hunk 2]
```diff
@@ -1201,6 +1202,9 @@ class boss_freya_summons : public CreatureScript
                     me->CastSpell(me->GetVictim(), SPELL_NATURE_FURY, false);
                     events.RepeatEvent(14000);
                     break;
+                case EVENT_ANCIENT_CONSERVATOR_GRIP:
+					me->CastSpell(me, SPELL_CONSERVATOR_GRIP, true);
+					break;
```
Comment (wizzymore) [inline src/server/scripts/Northrend/Ulduar/Ulduar/boss_freya.cpp:1207]:
```suggestion
                case EVENT_ANCIENT_CONSERVATOR_GRIP:
                   me->CastSpell(me, SPELL_CONSERVATOR_GRIP, true);
                   break;
```
Same goes here

File: src/server/scripts/Northrend/Ulduar/Ulduar/boss_freya.cpp [hunk 3]
```diff
@@ -1334,4 +1338,4 @@ void AddSC_boss_freya()
     new achievement_freya_knock_on_wood("achievement_freya_knock_on_wood", 1);
     new achievement_freya_knock_on_wood("achievement_freya_knock_knock_on_wood", 2);
     new achievement_freya_knock_on_wood("achievement_freya_knock_knock_knock_on_wood", 3);
-}
+}
```
Comment (wizzymore) [inline src/server/scripts/Northrend/Ulduar/Ulduar/boss_freya.cpp:1341]: Remove the white space at the end

**Review by wizzymore — CHANGES_REQUESTED** [review]

File: src/server/scripts/Northrend/Ulduar/Ulduar/boss_freya.cpp (hunk 3 above)
Comment (55Honey) [reply src/server/scripts/Northrend/Ulduar/Ulduar/boss_freya.cpp:1341]: Not sure how to fix that, see #welcome-n-offtopic on discord if you can. There is no space in my file.

Comment (wizzymore) [reply src/server/scripts/Northrend/Ulduar/Ulduar/boss_freya.cpp]: My bad on this one, was because of the icon from github!

Comment (55Honey) [reply src/server/scripts/Northrend/Ulduar/Ulduar/boss_freya.cpp]: i think there was an issue indeed, locus313 said on discord i need a new line at the end of the file. after pushing that, the difference vanished :-)

**Review by Helias — APPROVED** [review]

**Review by wizzymore — APPROVED** [review]

**Comment by wizzymore** [comment]
Tested and works as it should

**Comment by FrancescoBorzi** [comment]
good job @55Honey and @wizzymore !

**Comment by pklloveyou** [comment]
Tested

**Comment by Kitzunu** [comment]
Thanks =)

**Pull request merged**